pandas/core/series.py

```diff
@@ -2576,10 +2576,135 @@ def _needs_reindex_multi(self, axes, method, level):
         """
         return False
 
-    @Appender(generic._shared_docs['align'] % _shared_doc_kwargs)
     def align(self, other, join='outer', axis=None, level=None, copy=True,
               fill_value=None, method=None, limit=None, fill_axis=0,
               broadcast_axis=None):
+        """
+        Align two Series on their axes with the specified join method.
+
+        The result is a tuple of (Series, type of other) with the same indexes.
+        Using `join`, one can specifiy if the union or intersection of the
+        indexes are retained or whether to keep the index of the first or
+        second Series.
+
+        Align supports MultiIndex, `level` can be used to select which index
+        that is used for alignment.
+
+        Parameters
+        ----------
+        other : DataFrame or Series
+            The Series object to align with.
+        join : {'outer', 'inner', 'left', 'right'}, default 'outer'
+            How to aggregate the indexes.
+        axis : axis : {0 or 'index', 1 or 'columns', None}, default None
+            Align on index (0), columns (1), or both (None).
+        level : int or level name, optional
+            Broadcast across a level, matching Index values on the
+            passed MultiIndex level.
+        copy : bool, default True
+            Always returns new objects. If False and no reindexing is
+            required then original objects are returned.
+        fill_value : scalar, default np.NaN
+            Value to use for missing values. Defaults to NaN, but can be any
+            "compatible" value.
+        method : str, default None
+            Method to use for filling holes in reindexed Series:
+                - pad / ffill: propagate last valid observation forward to next valid
```

Comment:
Lists in rst don't need an indentation level, but, you need to add a blank line above it